### PR TITLE
Adds defaultLocale + spokenLanguages

### LINKED ISSUE #50: Language declaration (0.5sp)
Part of #28 is "internationalization". Before a proper internationalization solution arrives, I wonder if we shouldn't at least declare the language of the whole resource in a dedicated field. (Otherwise, it seems hard to use.)

If we decided to do that, the question is where to put the declaration (in other words, what's "the resource"). Is it the whole hotel (i.e. it's data index)? Or is it a single off-chain document (e.g. description)?

### PR DESCRIPTION
**Pull request by JirkaChadima**
Adds two new fields and closes #50

- `defaultLocale` - locale of the texts such as `description` in all of hotel's data structures. It is named default in anticipation of proper localization which i presume will add list of all supported locales on this level. 
- `spokenLanguages` - list of languages that the hotel does speak

It is not added to airlines, because there is no larger text field that would need localization in the future. It also seems to me that `spokenLanguages` does not really apply to airlines either.

In order to add support for this throughout the platform, we further need to:
- [ ] show support for this in wt-fixtures
- [ ] bump swagger docs in wt-read-api + test support for this
- [ ] bump swagger docs in wt-write-api + test support for this
- [ ] show these fields in the hotel explorer

Non critical
- [ ] bump swagger docs in wt-search-api + test support for this
- [ ] add support for search by default locale in wt-search-api

### CODE REVIEW

File: hotel-data-swagger.yaml [hunk 1]
```diff
@@ -60,6 +60,10 @@ components:
         bookingUri:
           $ref: 'https://raw.githubusercontent.com/windingtree/wiki/ef3fd90df7fcd1461cad1be2cb5bfeefb4fe4633/shared-resources.yaml#/components/schemas/UriType'
           description: URI pointing to an instance of the WT Booking service.
+        defaultLocale:
+          $ref: 'https://raw.githubusercontent.com/windingtree/wiki/7369d8e296049eb85e3192eaebed5543a1627316/shared-resources.yaml#/components/schemas/LanguageCodeType'
+          description: Code of language in which all textual information for this record is presented. If no default locale is declared, all of the tooling
```
Comment (czervenka) [inline hotel-data-swagger.yaml]: It's not only our tooling who can use the data format. What about a wordage: "... If no default locale is declared, the texts are expected to be in English language."?

Comment (JirkaChadima) [reply hotel-data-swagger.yaml]: What about it? If this field is not mandatory, it has to have a defined behaviour when it's missing. We can of course pick Swahili as default but English makes sense to me. Maybe we can make it mandatory, but it's backwards incompatible.

Comment (czervenka) [reply hotel-data-swagger.yaml]: I'm not sure whether we understand each other now. I wanted to replace:

> If no default locale is declared, all of the tooling should silently presume English.

by

> If no default locale is declared, the texts are expected to be in English language.

Comment (czervenka) [reply hotel-data-swagger.yaml]: ... but it's nothing important. Leave it as it is, if you want to.

Comment (JirkaChadima) [reply hotel-data-swagger.yaml]: Ahh.. sorry, I misunderstood. Your wording is better, I'll replace it.

**Pull request merged**